flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/optimize/program/FlinkOptimizeContext.scala

```diff
@@ -30,6 +30,12 @@ trait FlinkOptimizeContext extends FlinkContext {
     */
   def getRexBuilder: RexBuilder
 
+  /**
+   * Gets the [[FlinkRelBuilder]] defined in [[org.apache.flink.table.api.TableEnvironment]]
+   * @return
+   */
+  def getFlinkRelBuilder: FlinkRelBuilder
```

Comment:
we can just replace `def getRexBuilder: RexBuilder` with `def getFlinkRelBuilder: FlinkRelBuilder`, because we can get RexBuilder via FlinkRelBuilder and `getRexBuilder` method is used only in one place.

Reply:
Done